sql/core/src/main/scala/org/apache/spark/sql/jdbc/H2Dialect.scala

```diff
@@ -72,6 +72,26 @@ private[sql] object H2Dialect extends JdbcDialect {
           assert(f.children().length == 2)
           val distinct = if (f.isDistinct) "DISTINCT " else ""
           Some(s"CORR($distinct${f.children().head}, ${f.children().last})")
+        case f: GeneralAggregateFunc if f.name() == "REGR_INTERCEPT" =>
+          assert(f.children().length == 2)
+          val distinct = if (f.isDistinct) "DISTINCT " else ""
+          Some(s"REGR_INTERCEPT($distinct${f.children().head}, ${f.children().last})")
+        case f: GeneralAggregateFunc if f.name() == "REGR_R2" =>
+          assert(f.children().length == 2)
+          val distinct = if (f.isDistinct) "DISTINCT " else ""
+          Some(s"REGR_R2($distinct${f.children().head}, ${f.children().last})")
+        case f: GeneralAggregateFunc if f.name() == "REGR_SLOPE" =>
+          assert(f.children().length == 2)
+          val distinct = if (f.isDistinct) "DISTINCT " else ""
+          Some(s"REGR_SLOPE($distinct${f.children().head}, ${f.children().last})")
+        case f: GeneralAggregateFunc if f.name() == "REGR_SXX" =>
```

Comment:
Do we ever hit this path? In `translateAggregate` you don't have `REGR_SXX` and in the PR description it is said that "...... REGR_SXX and REGR_SXY are replaced to other expression in runtime"? Actually `REGR_SXY` is not converted to other expression in runtime, right?

Reply:
Thank you. I forgot to remove it.